heft-plugins/heft-jest-plugin/src/HeftJestReporter.ts

```diff
@@ -65,11 +73,10 @@ export default class HeftJestReporter implements Reporter {
       this._terminal.write(Colors.greenBackground(Colors.black('PASS')));
     }
 
-    const duration: string = test.duration ? `${test.duration / 1000}s` : '?';
     this._terminal.writeLine(
-      ` ${this._getTestPath(
-        test.path
-      )} (duration: ${duration}, ${numPassingTests} passed, ${numFailingTests} failed)`
+      ` ${this._getTestPath(test.path)} (duration: ${
+        durationMs / 1000
```

Comment:
What does this formatting look like?

Reply:
I wish we could standardize this better across our projects.

I did some work on that a long time ago in https://github.com/microsoft/rushstack/commit/c8b39d46e6e55152e6df247ed63489651edf3d49 but never merged it.

Reply:
`PASS src\test\JestPlugin.test.ts (duration: 1.726s, 2 passed, 0 failed)` is how the output appears when running with these changes, though I will make a change to use `toFixed` so that we have a constant number of decimals.